packages/maskbook/src/plugins/Wallet/database/types.ts

```diff
@@ -13,29 +13,6 @@ export interface ERC20TokenRecord {
     symbol: string
 }
 
-export interface ERC721TokenRecord {
```

Comment:
Why remove it? We need ERC721 to store into DB.

Reply:
It's moved.

Reply:
Just replaced by ERC721TokenDetailed.